src/BizHawk.Client.Common/config/Config.cs

```diff
@@ -36,10 +36,18 @@ public class Config
 				new[] { CoreNames.Gambatte, CoreNames.Sameboy, CoreNames.GbHawk, CoreNames.SubGbHawk }),
 			(new[] { VSystemID.Raw.GBL },
 				new[] { CoreNames.GambatteLink, CoreNames.GBHawkLink, CoreNames.GBHawkLink3x, CoreNames.GBHawkLink4x }),
+			(new[] { VSystemID.Raw.GEN },
+				new[] { CoreNames.Gpgx, CoreNames.PicoDrive }),
+			(new[] { VSystemID.Raw.GG },
+				new[] { CoreNames.Gpgx, CoreNames.SMSHawk }),
 			(new[] { VSystemID.Raw.PCE, VSystemID.Raw.PCECD, VSystemID.Raw.SGX, VSystemID.Raw.SGXCD },
 				new[] { CoreNames.TurboNyma, CoreNames.HyperNyma, CoreNames.PceHawk }),
 			(new[] { VSystemID.Raw.PSX },
 				new[] { CoreNames.Octoshock, CoreNames.Nymashock }),
+			(new[] { VSystemID.Raw.SMS },
+				new[] { CoreNames.Gpgx, CoreNames.SMSHawk }),
+			(new[] { VSystemID.Raw.SG },
+				new[] { CoreNames.Gpgx, CoreNames.SMSHawk }),

```

Comment:
You can combine all these, as GB and PCE are.

Reply:
I tried that, but then you cannot individually select it per console.

Reply:
Is that desired? I thought the whole point was to use GPGX' more accurate emulation by default; most people won't be switching at all, and if they did I don't see why they'd want it to be separate for the sister consoles.